autonomy/cli/mint.py

```diff
@@ -94,3 +94,87 @@ def protocol(
         password=password,
         nft_image_hash=nft,
     )
+
+
+@mint.command()
+@package_path_decorator
+@key_path_decorator
+@password_decorator
+@dependencies_decorator
+@nft_decorator
+@pass_ctx
+def contract(
+    ctx: Context,
+    package_path: Path,
+    keys: Path,
+    password: Optional[str],
+    dependencies: Tuple[str],
+    nft: Optional[str],
+) -> None:
+    """Mint a contract component."""
+
+    mint_component(
+        package_path=package_path,
+        package_type=PackageType.CONTRACT,
+        keys=keys,
+        chain_type=cast(ChainType, ctx.config.get("chain_type")),
+        dependencies=list(map(int, dependencies)),
+        password=password,
+        nft_image_hash=nft,
+    )
+
+
+@mint.command()
+@package_path_decorator
+@key_path_decorator
+@password_decorator
+@dependencies_decorator
+@nft_decorator
+@pass_ctx
+def connection(
+    ctx: Context,
+    package_path: Path,
+    keys: Path,
+    password: Optional[str],
+    dependencies: Tuple[str],
+    nft: Optional[str],
+) -> None:
+    """Mint a connection component."""
+
+    mint_component(
+        package_path=package_path,
+        package_type=PackageType.CONNECTION,
+        keys=keys,
+        chain_type=cast(ChainType, ctx.config.get("chain_type")),
+        dependencies=list(map(int, dependencies)),
+        password=password,
+        nft_image_hash=nft,
+    )
+
+
+@mint.command()
+@package_path_decorator
+@key_path_decorator
+@password_decorator
+@dependencies_decorator
+@nft_decorator
+@pass_ctx
+def skill(
+    ctx: Context,
+    package_path: Path,
+    keys: Path,
+    password: Optional[str],
+    dependencies: Tuple[str],
+    nft: Optional[str],
+) -> None:
+    """Mint a skill component."""
+
+    mint_component(
```

Comment:
minor optional comment

probably it's possible to make a command argument type of PackageConnection,cause subcommands diff are only in this argument. just DRY